ext/smoke/steps/run-validation-tests.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+set -e
+set -x
```

Comment:
Should this be turned off before we promote?

Reply:
Addressed above.